src/models/Url.js

```diff
@@ -0,0 +1,44 @@
+//const { string } = require("joi");
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+let UrlSchema = new Schema(
+  {
+    user: {
+      type: Schema.Types.ObjectId,
+      ref: "user",
+    },
+    urlCode: {
+      type: String,
+      required: true,
+    },
+
+    longUrl: {
+      type: String,
+      required: true,
+    },
```

Comment:
These two fields need indexing since you are using them in querying the data. Fields that are used in queries need indexing to speedup the query irrespective of the DB technology.